source/command/analyzeCommand.ml

```diff
@@ -305,6 +312,19 @@ let with_performance_tracking ~debug ~f =
   result
 
 
+let verify_taint_configuration ~static_analysis_configuration () =
+  try TaintAnalysis.initialize_configuration ~static_analysis_configuration with
+  | Taint.TaintConfiguration.TaintConfigurationError _ as exn -> raise exn
+  | exn ->
+      (* The backtrace is lost if the exception is caught at the top level, because of `Lwt`. *
+         Let's print the exception here to ease debugging. *)
+      Log.log_exception
+        "Failed to validate taint configuration"
+        exn
+        (Worker.exception_backtrace exn);
+      raise exn
```

Comment:
Is this copy/pasted from somewhere? Can we avoid code duplication, maybe with a function?

Reply:
It was copy pasted - yes, moving it with-in the scheduler fixes this issue.